CMakeLists.txt

```diff
@@ -222,6 +214,29 @@ if(WIN32)
     DESTINATION ${INSTALLDIR}/styles/
   )
 
+  set(CPACK_GENERATOR "NSIS")
+  set(CPACK_PRE_BUILD_SCRIPTS "${CMAKE_SOURCE_DIR}/CMake/WindowsPreBuild.cmake")
+  set(CPACK_PACKAGE_INSTALL_DIRECTORY "AGAVE ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")

```

Comment:
Should all the AGAVE version install to different folders, or should they all share the same on?

If someone has an existing installation of AGAVE, is the installer smart enough to delete the old installation? Will there be conflicts in the `agave://` url protocol registration if a new version of AGAVE is installed?

Reply:
currently this will always overwrite the url protocol handler with the most recently installed agave.

Reply:
On windows, we don't automatically uninstall previous agave. It's true that they could mess up the protocol handler by installing a new version and then uninstalling the old.  I'll think about how best to handle that.  At least we can warn them to uninstall FIRST before installing.

Reply:
Maybe I could just make the reg key have the version number in it.  will have to test